mindsdb/integrations/handlers/vertex_handler/vertex_client.py

```diff
@@ -0,0 +1,90 @@
+from mindsdb.utilities import log
+from google.cloud import aiplatform
+from google.oauth2 import service_account
+import pandas as pd
+import json
+
+
+class VertexClient:
+    """A class to interact with Vertex AI"""
+
+    def __init__(self, credentials_path, project_id):
+        credentials = service_account.Credentials.from_service_account_file(credentials_path)
+        aiplatform.init(
+            project=project_id,
+            location="us-central1",
+            staging_bucket="gs://my_staging_bucket",
+            credentials=credentials,
+            # the name of the experiment to use to track
+            # logged metrics and parameters
+            experiment="my-experiment",
+            # description of the experiment above
+            experiment_description="my experiment description",
```

Comment:
I think location, staging_bucket, experiment also should be parameterized

Reply:
Good spot!